peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -51,6 +51,11 @@ <h1>{{ document.title }}</h1>
                  target="_blank">
                 <i class="bi bi-linkedin linkedin-forecolor share-icon"></i>
               </a>
+              <a href="mailto:?subject={% trans 'Take a look at this document from' %} {{ APP_NAME }}: {{ document.title|truncatewords:7|escape }}&body={{ request.build_absolute_uri }}"
```

Comment:
What if, in addition to the href, you put the subject in as `data-subject=...` and the body similarly, and then the vue code just fetches those attributes, rather than having to re-compute it in Vue? That way the translation all happens through Django.

Reply:
This is a much better way, thank you. Also, I don't think we'll need to add the body as an attribute since it's different in both cases. Right now, I'm using the _combined_ computed value as the body for the **ShareSelectionModal**
This is what it looks like now
![Screenshot 2025-01-17 at 11 28 57](https://github.com/user-attachments/assets/20c97399-ef86-48bc-b23e-ac91516a1112)